subsys/net/lib/config/Kconfig

```diff
@@ -206,7 +206,7 @@ endif # NET_DHCPV6
 
```

Comment:
Couple of things changes in the commit message. Please change the subject to
`net: config: Fix sntp clock init Kconfig depends`
Also the commit body needs text, write there about why this change is needed.

Reply:
Applied the required changes